libs/ui-lib/lib/common/components/clusterConfiguration/UploadSSH.tsx

```diff
@@ -27,15 +27,6 @@ const UploadSSH: React.FC<UploadSSHProps> = ({ isRequired, labelText }) => {
       name={name}
       helperText={<SshPublicKeyHelperText />}
       idPostfix="discovery"
-      onBlur={() => value && setValue(trimSshPublicKey(value))}
```

Comment:
We don't trim the value on blur anymore. is this expected ?

Reply:
I changed it.
Now I'm doing this inside libs/ui-lib/lib/common/components/ui/formik/UploadField.tsx in line 76 and 84